src/main/java/org/truffleruby/options/CommandLineParser.java

```diff
@@ -443,9 +436,9 @@ private void processArgument() {
                             break;
                         }
                     }
-                    throw new MainExitException(1, "truffleruby: unknown option " + argument);
+                    throw new CommandLineException("unknown option " + argument);
                 default:
-                    throw new MainExitException(1, "truffleruby: unknown option " + argument);
+                    throw new CommandLineException("unknown option " + argument);
```

Comment:
MRI provides some info on how to list available options which is nice: E.g.  `invalid option -b  (-h will show valid options) (RuntimeError)`